crates/bevy_window/src/window.rs

```diff
@@ -641,8 +667,8 @@ pub struct WindowDescriptor {
     /// May vary from the physical height due to different pixel density on different monitors.
     pub height: f32,
     /// The position on the screen that the window will be centered at.
-    /// If set to `None`, some platform-specific position will be chosen.
-    pub position: Option<Vec2>,
+    /// See [`WindowPosition`] for available values.
```

Comment:
this line is not very useful, cargo doc will already link to the enum on the field type

Reply:
I wanted to explicitly point to `WindowPosition` page, because just the name "WindowPosition" doesn't convey that it could be something else than just raw coordinates (at least in my opinion). But maybe you're right that this is unnecessary. Or maybe different wording/naming would fit this better?

Reply:
Yeah, maybe something like that would be clearer. But TBF I'm probably too involved(?) to make an objective judgement here, so I'll rely on other people's opinion.

Reply:
I like `WindowLocation`; I think that would help communicate that there's multiple subtle variants.